### DISCUSSION TITLE
Postgres Anonymizer

### DISCUSSION THREAD

**Discussion by jdu** [hydradatabase/hydra#87] [Ideas]
Given this is a project to provide a super-duper postgres warehouse with columnar data storage which is more or less geared at analytics... 

One of the things that's *sorely* missing from services like RDS/Aurora and others, and is available in solutions like Snowflake is masking functions. Postgresql Anonymizer (https://postgresql-anonymizer.readthedocs.io/en/latest/) is a monster implementation of it, and it makes it more feasible to do something that's really complicated alot of the time in stock postgres - hide or mask data from specific roles and users based on some rules. I'm not 100% sure it'll work side by side with the columnar extension, but it would be very welcome if it did.

The docker build in here is a bit cryptic, but if there's some pointers to adding a new extension to the build I would be happy to take a crack at it!

**Comment by jdu** [comment]
postgresml is another one that I would be keen to see in here.

**Comment by JerrySievert** [comment]
we've discussed postgres anonymizer internally - I'm a big fan of it.  no updates to add, yet, but stay tuned.
